File: DemoApp/DemoApp/NativeStoryView.m
//
//  NativeStoryView.m
//  DemoApp
//
//  Created by Glispa GmbH on 12/29/15.
//  Copyright © 2015 Glispa GmbH All rights reserved.
//

#import "NativeStoryView.h"

@implementation NativeStoryView


+ (CGFloat)desiredHeight {
    return 160;
}


#pragma mark - Overriden methods


// Implement + (NSString*)xibName only if this class is related with XIB
+ (NSString *)xibName {
    return NSStringFromClass([self class]);
}

@end

File: DemoApp/DemoAppSwift/NativeStoryView.swift
//
//  NativeStoryView.swift
//  DemoApp
//
//  Created by Glispa GmbH on 19/05/16.
//  Copyright © 2016 Glispa GmbH. All rights reserved.
//

import UIKit
import AvocarrotNativeView

class NativeStoryView: AVONativeView {

    static func desiredHeight() -> CGFloat {
        return 160.0
    }

    override static func xibName() -> String {
        return NSStringFromClass(self)
    }

}
